tests/base.py

```diff
@@ -34,28 +34,51 @@ def __init__(self):
 
 class CleanModel():
     def __init__(self):
+        self.user_name = None
         self.controller = None
+        self.controller_name = None
         self.model = None
+        self.model_name = None
+        self.model_uuid = None
 
     async def __aenter__(self):
         self.controller = Controller()
-        await self.controller.connect_current()
+        juju_data = JujuData()
```

Comment:
Same question as above.

Reply:
This is to fix the test failing with `KeyError`, e.g. https://travis-ci.org/juju/python-libjuju/jobs/281013483

When the model is added with `Controller.add_model`, the juju data isn't updated.  But since a few of the tests use `connect_current`, it was trying to read the data about the current model.  We patched in a portion of that, but didn't have the model UUID in there.  TBH, I'm not sure how those tests ever worked, or at what point they broke.

Some of the tests affected by this could be cleaned up by just getting the connection from the `CleanModel` directly, but at least the test for `connect_current` itself requires this.